x-pack/plugins/lens/public/datasources/form_based/field_item.tsx

```diff
@@ -239,9 +239,9 @@ export const InnerFieldItem = function InnerFieldItem(props: FieldItemProps) {
                   field: value.field.name,
                 },
               })
-            : i18n.translate('xpack.lens.indexPattern.moveToWorkspaceDisabled', {
+            : i18n.translate('xpack.lens.indexPattern.moveToWorkspaceNotAvailable', {
                 defaultMessage:
-                  "This field can't be added to the workspace automatically. You can still use it directly in the configuration panel.",
+                  'To visualize this field, drag it directly to the layer pane. Drag-and-drop to the workspace is unsupported.',
```

Comment:
Would it make sense to remove all references to "dragging" in this messaging (as that is not the only way users can add fields to layers or the workspace in Lens)?

```suggestion
                  'To visualize this field, please add it directly to the desired layer. Adding this field to the workspace is not supported based on your current configuration.',
```

Reply:
Thanks, @MichaelMarcialis !